<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdrDeviceBladerfRxctl</class>
 <widget class="QWidget" name="SdrDeviceBladerfRxctl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="gainModeLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Gain mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gainModeCombo">
       <property name="toolTip">
        <string>Select gain mode</string>
       </property>
       <item>
        <property name="text">
         <string>Manual</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="gainLabel">
       <property name="toolTip">
        <string>Overall system gain</string>
       </property>
       <property name="text">
        <string>System gain</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="gainValueLabel">
       <property name="toolTip">
        <string>Overall system gain</string>
       </property>
       <property name="text">
        <string>0 dB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="gainSlider">
       <property name="toolTip">
        <string>Overall system gain</string>
       </property>
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="bias1Box">
       <property name="text">
        <string>RX1 bias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bias2Box">
       <property name="text">
        <string>RX2 bias</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>131</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
